code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -32,7 +32,7 @@ BONUS
 			if(1, 2, 3)
 				to_chat(M, "<span notice='notice'>[pick("You swallow excess mucus.", "You lightly cough.")]</span>")
 			else
-				M.emote("cough")
+				M.emote("cough", , , 1)
```

Comment:
You should be using named arguments in these situations, not adding commas, because if someone changes the arg order or adds an arg this breaks instantly. It's also much more difficult to read.